README.md

```diff
@@ -133,7 +133,8 @@ As of writing this README, the following methods are supported:
 - `set_transaction_tags` - sets the tags on a transaction
 - `set_budget_amount` - sets a budget's value to the given amount (date allowed, will only apply to month specified by default). A zero amount value will "unset" or "clear" the budget for the given category.
 - `create_manual_account` - creates a new manual account
-- `upload_account_balance_history` - uploads account history csv file for a given account
+- `delete_account` - deletes an account by the provided account id
+- `upload_and_parse_balance_history` - uploads and parses account history csv file for a given account
```

Comment:
I'd vote for keeping the existing method rather than make a breaking change. The fact that the API now requires us to poll for the result is an implementation detail.

Reply:
Thanks @andrecloutier. That's a fair point. I did a bunch of refactoring to keep the existing method `upload_account_balance_history`.